platform-includes/sourcemaps/overview/javascript.mdx

```diff
@@ -16,6 +16,14 @@ This guide assumes you are using a Browser JavaScript SDK. For instructions on h
 
 If you want to configure source maps to upload manually, follow the guide for your bundler or build tool below.
 
+<PlatformSection notSupported={["javascript.capacitor", "javascript.cordova"]}>
+<Alert>
```

Comment:
Not sure I like this note. Feels out of place honestly.

Reply:
Hmm I am very open to rephrase this, but IMHO it does make sense to mention here what all the things we are talking about on this page and subpages actually does (=inject debug ids into your build files)?

Reply:
Actually you're right probably